README.md

```diff
@@ -21,6 +21,14 @@
   - depending on your implementation, you may need to add [a correlation variable](https://docs.camunda.io/docs/components/modeler/bpmn/message-events/#messages) to the instance pre migration
 - Boundary events:
   - currently not supported
+- Call Activity
+   - To migrate a subprocess that is started from a call activity, the migrator must set the `legacyId` variable for the subprocess. This requires propagating the parent variables. This can be achieved by updating the C8 call activity in one of the following ways:  
```

Comment:
Question: are both instances truly migrated or are we just migrating the parent instance and then C8 automatically creates the child instance and it moves naturally through the subprocess?

Reply:
We only create the root process, the subprocess is created by C8.
My understanding is:
- We create the parent issue via C8 API
- We activate the parent issue's call activity -> this makes C8 create the subprocess
- We  activate the subprocess